db/crud.go

```diff
@@ -1282,7 +1282,7 @@ func (db *Database) updateAndReturnDoc(
 	}
 
 	// Now that the document has successfully been stored, we can make other db changes:
-	base.InfofCtx(db.Ctx, base.KeyCRUD, "Stored doc %q / %q", base.UD(docid), newRevID)
+	base.InfofCtx(db.Ctx, base.KeyCRUD, "Stored doc %q / %q as #%v", base.UD(docid), newRevID, docSequence)
```

Comment:
Is `docSequence` correct here, or should I be using `doc.Sequence`?

Reply:
I believe they should always be the same at this point, but it's probably more future-proof to use doc.Sequence.